content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,223 @@
+|`NodeLabelPriority`               |   N   |              |  `>=1.0`       |Checks whether a particular label exists on a node or not, regardless of its value. <br>If presence is true, prioritizes nodes that have the specified label, regardless of value. <br>If presence is false, prioritizes nodes that do not have the specified label.|
+|`NodePreferAvoidPodsPriority`     |   Y   |       10000  |  `>=1.4`       |Priorities nodes according to the node annotation scheduler.alpha.kubernetes.io/preferAvoidPods.|
+|`RequestedToCapacityRatioPriority`|   N   |              |  `>=1.11`      |Creates a requestedToCapacity based ResourceAllocationPriority using default resource scoring function shape. <br>The default function assigns 1.0 to resource when all capacity is available and 0.0 when requested amount is equal to capacity.|
+|`ResourceLimitsPriority`          |   N   |              |  `>=1.13`       |Increases score of input node by 1 if the node satisfies input pod's resource limits. <br>In detail, this priority function works as follows, <br>If a node does not publish its allocatable resources (cpu and memory both), the node score is not affected. <br>If a pod does not specify its cpu and memory limits both, the node score is not affected. <br>If one or both of cpu and memory limits of the pod are satisfied, the node is assigned a score of 1. <br>Rationale of choosing the lowest score of 1 is that this is mainly selected to break ties between nodes that have same scores assigned by one of least and most requested priority functions.|
+|`SelectorSpreadPriority`          |   Y   |       1      | `>=1.1`        |Spreads pods across hosts, considering pods belonging to the same service,RC,RS or StatefulSet. <br>When a pod is scheduled, it looks for services, RCs,RSs and StatefulSets that match the pod, then finds existing pods that match those selectors. <br>It favors nodes that have fewer existing matching pods. <br>i.e. it pushes the scheduler towards a node where there's the smallest number of pods which match the same service, RC,RSs or StatefulSets selectors as the pod being scheduled.|
+|`ServiceSpreadingPriority`        |   N   |              |  `>=1.0`       |Spreads pods by minimizing the number of pods (belonging to the same service) on the same node. Largely replaced by "SelectorSpreadPriority", backward compatibility with 1.0.|
+|`TaintTolerationPriority`         |   Y   |       1      |  `>=1.3`       |Prepares the priority list for all the nodes based on the number of intolerable taints on the node|
+
+## User Cases
+
+### checkServiceAffinity
+
```

Comment:
Some copyedits needed here, but this is definitely useful content at a concept level. I'll copyedit after we figure out reorganizing the rest of the content.

Reply:
Thanks in advance!

Reply:
Yes, I have the same feeling.